files/build_templates/docker_image_ctl.j2

```diff
@@ -163,6 +160,12 @@ start() {
 
 {%- if docker_container_name == "database" %}
     echo "Creating new {{docker_container_name}} container"
+    # database_config.json contains the information how to create database instances and distribute the database on instances
+    # if database_config exists in old_config, use it; otherwise use the default one in new image
+    if [ -f /etc/sonic/old_config/database_config.json ]; then
+        echo "Use database_config.json from old system..."
+        cp /etc/sonic/old_config/database_config.json /etc/sonic/
```

Comment:
>cp [](start = 8, length = 2)

We need also backup running config in warm-reboot script.

Reply:
this will be done when adding warm reboot support